twilio_integration/public/js/voice_call_settings.js

```diff
@@ -0,0 +1,34 @@
+
+frappe.ui.form.on('Voice Call Settings', {
+	onload: function(frm) {
+		if (!frappe.user_roles.includes('System Manager')){
+			frm.set_value('user', frappe.session.user)
+		}
+	},
+	refresh: function(frm) {
+		frappe.call({
+			method: "twilio_integration.twilio_integration.api.get_twilio_phone_numbers",
+			callback: function(resp) {
+				if (resp.message.length) {
+					frm.set_df_property('twilio_number', 'options', resp.message);
+					frm.refresh_field('twilio_number');
+				}
+				else {
+					frappe.show_alert({
+						message:__('Voice settings rely on twilio settings. Please make sure that twilio settings are configured & enabled.'),
```

Comment:
`Twilio` (instead of `twilio`) in the message?

Reply:
Done!